packages/web-react/src/components/TextField/README.md

```diff
@@ -46,23 +46,23 @@ TextField with password toggle (button to reveal the password):
-| `pattern`           | `string`                                                      | -       | no       | Defines regular expressions for allowed value types                     |
-| `placeholder`       | `string`                                                      | -       | no       | Input placeholder                                                       |
-| `ref`               | `ForwardedRef<HTMLInputElement>`                              | -       | no       | Input element reference                                                 |
-| `type`              | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | -       | no       | Input type                                                              |
-| `validationState`   | [Validation dictionary][dictionary-validation]                | -       | no       | Type of validation state                                                |
-| `validationText`    | `string`, `string[]`                                          | -       | no       | Validation text                                                         |
-| `value`             | `string`                                                      | -       | no       | Input value                                                             |
+| `autoComplete`      | `string`                                                      | –       | no       | [Automated assistance in filling][autocomplete-attr]                    |
+| `hasPasswordToggle` | `boolean`                                                     | –       | no       | If true, the `type` is set to `password` and a password toggle is shown |
+| `helperText`        | `string`                                                      | –       | no       | Custom helper text                                                      |
+| `id`                | `string`                                                      | –       | yes      | Input and label identification                                          |
+| `inputWidth`        | `number`                                                      | –       | no       | Input width                                                             |
+| `isDisabled`        | `boolean`                                                     | –       | no       | Whether is field disabled                                               |
+| `isLabelHidden`     | `boolean`                                                     | –       | no       | Whether is label hidden                                                 |
+| `isRequired`        | `boolean`                                                     | –       | no       | Whether is field required                                               |
+| `label`             | `string`                                                      | –       | yes      | Label text                                                              |
```

Comment:
The label is not technically required.

Reply:
Technically not, but we want it to be filled every time. :)

Reply:
Is this a fix or BC?

Reply:
@adamkudrna This is React and if the `label` is marked as required it should be the same in the types.